tests/sles4sap/qesapdeployment/configure.pm

```diff
@@ -22,7 +22,6 @@ sub run {
     my $qesap_provider = lc get_required_var('PUBLIC_CLOUD_PROVIDER');
 
     my %variables;
-    $variables{PROVIDER} = $qesap_provider;
```

Comment:
Is this really needed?

Reply:
It is no more needed line of code as the config.yaml no more has PROVIDER  as configurable variable. So I deleted the no more needed line of code.
Having it in the code is not harmful but just no more needed.